### ISSUE TITLE
Enhancement - Support UUID

### ISSUE THREAD

**Issue by hieulw** [xnuinside/omymodels#9]
Hi again, seem like it doesn't support UUID
```sql
CREATE TABLE "prefix--schema-name"."table" (
  _id uuid PRIMARY KEY,
);
```
will be convert to
```python
from gino import Gino
db = Gino(schema="prefix--schema-name")

class Table(db.Model):
    __tablename__ = 'table'

    _id = db.Column(OMM_UNMAPPED_TYPE(), nullable=False, primary_key=True)
```
Expected:
```python
from sqlalchemy.dialects.postgresql import UUID
from gino import Gino

db = Gino()

class Table(db.Model):
    __tablename__ = 'table'
    __table_args__ = dict(schema="prefix--schema-name")

    _id = db.Column(UUID, primary_key=True)
```
Notes: the schema may be different between tables, should place in model definition instead of gino instance

**Comment by xnuinside** [comment]
@hieulw , again thanks for very useful issues, I will do it tomorrow

**Comment by xnuinside** [comment]
UUID - added, comment in this issue - https://github.com/xnuinside/omymodels/issues/10, about don't show nullable=False in columns with primary_key=True - will be in next commits & schema in table_args

**Comment by xnuinside** [comment]
@hieulw  hello! I released 0.5.0 version with all changes, example in tests https://github.com/xnuinside/omymodels/blob/main/tests/test_models_simple.py#L117
